sign.go

```diff
@@ -339,6 +348,45 @@ func (si *signerInfo) SetUnauthenticatedAttributes(extraUnsignedAttrs []Attribut
 	return nil
 }
 
+type TimestampTokenRequestCallback func(digest []byte) ([]byte, error)
+
+func (sd *SignedData) RequestSignerTimestampToken(signerID int, callback TimestampTokenRequestCallback) error {
+	if len(sd.sd.SignerInfos) < (signerID + 1) {
+		return fmt.Errorf("no signer information found for ID %d", signerID)
+	}
+
+	if callback == nil {
+		return fmt.Errorf("no callback defined")
+	}
+
+	tst, err := callback(sd.sd.SignerInfos[signerID].EncryptedDigest)
+	if err != nil {
+		return err
+	}
+	return sd.AddTimestampTokenToSigner(signerID, tst)
+}
+
+// AddTimestampTokenToSigner inserts TimestampToken which described in RFC3161 into
+// unauthenticated attribute of that signer.
```

Comment:
define what `signerID` is ? and `tst` ?

Reply:
Updated at 3807938c83f951ee5425c3fc528f87543537f443